corehq/apps/user_importer/helpers.py

```diff
@@ -149,14 +145,12 @@ def _include_user_data_changes(self):
 class CommCareUserImporter(BaseUserImporter):
     def update_password(self, password):
         self.user.set_password(password)
-        self.logger.add_change_message(_("Password Reset"))
+        self.logger.add_change_message("Password Reset")
```

Comment:
was already present
https://github.com/dimagi/commcare-hq/commit/c7e498725df7a04c5b8133fbbaa750c6fbe3486b#diff-c9557c19413e04ae10a25832b30f73ee68d4e9bdae31b49f5404b9086b80f126R16207

Reply:
Not sure why the en file has
```
msgid "Password Reset"
msgstr "Manejo de Casos"
```

Reply:
en
```
#: corehq/apps/domain/templates/login_and_password/password_reset_form.html:7
#: corehq/apps/domain/urls.py:87 corehq/apps/user_importer/helpers.py:152
#, fuzzy
msgid "Password Reset"
msgstr "Manejo de Casos"
```

es
```

#: corehq/apps/domain/templates/login_and_password/password_reset_form.html:7
#: corehq/apps/domain/urls.py:87 corehq/apps/user_importer/helpers.py:152
msgid "Password Reset"
msgstr "Contraseña Restablecida"
```

fra
```
#: corehq/apps/domain/templates/login_and_password/password_reset_form.html:7
#: corehq/apps/domain/urls.py:87 corehq/apps/user_importer/helpers.py:152
msgid "Password Reset"
msgstr "Réinitialisation du mot de passe"
```

no translations in other langs